src/Arcus.Messaging.Abstractions/MessageHandling/MessageRouter.cs

```diff
@@ -263,6 +263,7 @@ protected async Task RouteMessageAsync<TMessageContext>(
             }
         }
 
+        [Obsolete("Will be removed in v3.0, as the entire fallback message handler structure becomes unnecessary")]
         private async Task<bool> TryProcessMessageAsync<TMessageContext>(
```

Comment:
Sure that this method is obsolete as well ?  Given the name, I wouldn't suspect that it has something to do with the fallback mechanism ?

Reply:
The callee of this method is already marked deprecated. This additional deprecation was made since this method is calling fallback functionality in its body.